mediawiki-containers

```diff
@@ -175,7 +188,11 @@ install() {
     check_out_mediawiki_containers
 
     cd $BASEDIR
-    
+
+    if [ "$platform" != "debian" -a "$platform" != "ubuntu" ]; then
```

Comment:
@gwicke install is not supported on OSX at all (yet), only start/stop is supported